common-util/functions/proposal.ts

```diff
@@ -0,0 +1,54 @@
+import { ethers } from 'ethers';
+import { isNil } from 'lodash';
+
+import { ethersToWei, formatToEth } from 'common-util/functions';
+import { ModuleDetails } from 'store/types';
+import { VEOLAS_QUORUM } from 'util/constants';
+
+// 2 million veolas in wei
+const quorumInWei = ethersToWei(`${VEOLAS_QUORUM}`);
+
+/**
+ * check if the current proposal has enough veOLAS to be executed
+ */
+export const getCurrentProposalInfo = (
+  proposal: ModuleDetails['scheduled_tweet']['tweets'][number],
+) => {
+  // example of voters: [ { '0x123': '1000000000000000000000000' } ]
+  const totalVeolasInWei = proposal?.voters?.reduce((acc, voter) => {
```

Comment:
If `proposal?.voters` is undefined, then `totalVeolasInWei` will also be undefined, right?
```suggestion
  const totalVeolasInWei = proposal?.voters ? proposal.voters..reduce((acc, voter) => {
```
else 0n.

Reply:
it shouldn't be undefined - would be an empty array, will remove the optional chaining